code/WorkInProgress/artemis/GeneratePlanets.dm

```diff
@@ -243,6 +243,42 @@ var/global/datum/planetManager/PLANET_LOCATIONS = new /datum/planetManager()
 	planet_area.name = name
 	region.clean_up(main_area=planet_area)
 
+	//Parallax it?
+	if(istype(generator, /datum/map_generator/snow_generator) && prob(15) )
+		var/angle = rand(110,250)
+		var/scroll_speed = rand(50, 100)
+		var/color_alpha = rand(30,60)/100
+		var/color_matrix = list(
+								1, 0, 0, color_alpha,
+								0, 1, 0, color_alpha,
+								0, 0, 1, color_alpha,
+								0, 0, 0, 1,
+								0, 0, 0, -1)
+		planet_area.area_parallax_layers = list(
+		/atom/movable/screen/parallax_layer/foreground/snow=list(color=color_matrix, scroll_speed=scroll_speed, scroll_angle=angle),
+		/atom/movable/screen/parallax_layer/foreground/snow/sparse=list(color=color_matrix, scroll_speed=scroll_speed+25, scroll_angle=angle),
+		)
+	else if(istype(generator, /datum/map_generator/desert_generator)&& prob(15) )
+		var/angle = rand(110,250)
+		var/scroll_speed = rand(75, 175)
+		var/color_alpha = rand(40,80)/100
+		var/color_matrix = list(
+								1, 0, 0, color_alpha,
+								0, 1, 0, color_alpha,
+								0, 0, 1, color_alpha,
+								0, 0, 0, 1,
+								0, 0, 0, -1)
+		planet_area.area_parallax_layers = list(
+			/atom/movable/screen/parallax_layer/foreground/dust=list(color=color_matrix, scroll_speed=scroll_speed, scroll_angle=angle),
+			/atom/movable/screen/parallax_layer/foreground/dust/sparse=list(color=color_matrix, scroll_speed=scroll_speed*1.5, scroll_angle=angle)
+		)
+
+	// Occlude overlays on edges
+	if(planet_area.area_parallax_layers)
+		for(var/turf/cordon/CT in planet_area)
```

Comment:
Aren't cordons in their own area?

Reply:
I didn't want to add additional processing where it wasn't necessary, but that does seem like a much better solution.

Reply:
They are, but such a snowflake check is required to permit full-tile occlusion overlays, so to prevent visual artefacts.

![image](https://github.com/goonstation/goonstation/assets/88833499/4c554bac-db9b-4147-b542-da18226b32f4)

Reply:
Okay, that solution doesn't really work as I kind of like the cordon's being part of the allocated area (at least at the moment).  Reverted to be snowflake. 😞